dev_tools/scripts/db_integrity_test.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+docker exec db psql -U postgres -d test -c "INSERT INTO local_ega.main(submission_file_path,submission_user,submission_file_extension,status,encryption_method) VALUES('test.c4gh','dummy','c4gh','INIT','CRYPT4GH');"
+
+docker container run --rm -i --name pg-backup --network=host $(docker build -f dev_tools/Dockerfile-backup -q .) /bin/sda-backup --action pg_basebackup
+
+DBCOPY=$(s3cmd -c dev_tools/s3conf ls s3://dumps/ | grep ".tar" | cut -d '/' -f4)
+
+docker container run -v $(pwd)/tmp:/home --rm -i --name pg-backup --network=host $(docker build --build-arg USER_ID=$(id -u) -f dev_tools/Dockerfile-backup -q .) /bin/sda-backup --action pg_bb-restore --name "$DBCOPY"
+
+docker exec -i db /bin/sh -c "rm -r data/pgdata"
+
+docker cp tmp/db-backup/ db:data/pgdata
```

Comment:
Why is this run *after* the restore command?

Reply:
It is not an actual restore with this command but it makes the pg data ready in order to run the postgres by pointing to that data or just copy them directly in the pg container as it is done in the test for simplicity